articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -22,78 +22,90 @@ ms.custom: aaddev
 
 [!INCLUDE [active-directory-develop-applies-v2](../../../includes/active-directory-develop-applies-v2.md)]
 
-In this quickstart, you'll learn how to integrate a Java web application with the Microsoft identity platform. Your app 
-will sign in a user, get an access token to call the Microsoft Graph API, and make a request to the Microsoft Graph API. 
+In this quickstart, you'll learn how to integrate a Java web application with the Microsoft identity platform. Your app will sign in a user, get an access token to call the Microsoft Graph API, and make a request to the Microsoft Graph API.
 
-When you've completed the guide, your application will accept sign-ins of personal Microsoft accounts (including outlook.com,
- live.com, and others) and work or school accounts from any company or organization that uses Azure Active Directory.
+When you've completed the guide, your application will accept sign-ins of personal Microsoft accounts (including outlook.com,live.com, and others) and work or school accounts from any company or organization that uses Azure Active Directory.
```

Comment:
Maybe: "When you've completed this quickstart, you'll have an app that can sign in Microsoft accounts such as outlook.com and live.com, as well as work or school accounts for any company or organization that uses Azure Active Directory.